test/ConfigTest.php

```diff
@@ -334,4 +334,19 @@ public function testInjectMultipleDelegators()
             $service->injected
         );
     }
+
+    public function testInvokablesWithoutAlias()
+    {
+        $dependencies = [
+            'invokables' => [
+                TestAsset\Service::class,
```

Comment:
Does this work in other containers at this time?

In zend-servicemanager currently, it doesn't; omitting the key means an alias is created, regardless of whether that alias is an integer. From what I can read of zend-auradi-config, it will not handle this, either.